internal/observability/event/eventer.go

```diff
@@ -142,21 +166,39 @@ func NewEventer(log hclog.Logger, c EventerConfig, opt ...Option) (*Eventer, err
 		return nil, fmt.Errorf("%s: failed to register json node: %w", op, err)
 	}
 
+	if opts.withSerializationLock == nil {
+		opts.withSerializationLock = new(sync.Mutex)
```

Comment:
Rather than do this I'd honestly just make it required. If we don't have a serialization lock we are at real risk of something going wonky with any legacy or fallback logger code.

Reply:
Agreed and this will make all the tests more explicit as well.  ty!